cmd/uplink/cmd/cp.go

```diff
@@ -39,166 +35,6 @@ func init() {
 	expires = cpCmd.Flags().String("expires", "", "optional expiration date of an object. Please use format (yyyy-mm-ddThh:mm:ssZhh:mm)")
 }
 
-// upload transfers src from local machine to s3 compatible object dst
-func upload(ctx context.Context, src fpath.FPath, dst fpath.FPath, showProgress bool) (err error) {
```

Comment:
Try to avoid moving and changing things in the same PR because it's unclear currently whether this func changed. (Of course sometimes it's unavoidable doing both.)

Reply:
i reverted those changes that just moved things.